opm/parser/eclipse/IntegrationTests/IOConfigIntegrationTest.cpp

```diff
@@ -298,12 +298,7 @@ BOOST_AUTO_TEST_CASE( NorneResttartConfig ) {
     rptConfig.push_back( std::make_tuple(240 , true , boost::gregorian::date( 2006,10,1)) );
     rptConfig.push_back( std::make_tuple(241 , true , boost::gregorian::date( 2006,10,10)) );
 
-
-    ParseContext parseContext;
-    ParserPtr parser(new Parser());
-    DeckConstPtr deck = parser->parseFile("testdata/integration_tests/IOConfig/RPTRST_DECK.DATA", parseContext);
-    EclipseState state( deck , parseContext );
-
+    auto state = Parser::parse("testdata/integration_tests/IOConfig/RPTRST_DECK.DATA");
```

Comment:
Nothing wrong, but the main reason I chose `parse` was because there are already functions with the names you mention, with the same input parameters, but different return value.  One possibility would be to hijack these names, but I think they are very widely used; `DeckPtr parseFile` and `DeckPtr parseString` etc.